bodhi/tests/server/test_models.py

```diff
@@ -361,6 +361,62 @@ def test_same_build_types(self):
         # This should not raise any Exception.
         self.db.flush()
 
+    @mock.patch('bodhi.server.util.requests.get')
+    def test_get_pkg_committers_from_pagure(self, mock_get):
+        """ Ensure that the package commiters can be found using the Pagure
+        API.
+        """
+        mock_get.return_value.status_code = 200
+        json_one = {
+            "access_groups": {
+                "admin": [],
+                "commit": ['patriots'],
+                "ticket": []
+            },
+            "access_users": {
+                "admin": [],
+                "commit": [],
+                "owner": [
+                    "mprahl"
+                ],
+                "ticket": ["jsmith"]
+            },
+            "close_status": [],
+            "custom_keys": [],
+            "date_created": "1494947106",
+            "description": "Python",
+            "fullname": "rpms/python",
+            "id": 2,
+            "milestones": {},
+            "name": "python",
+            "namespace": "rpms",
+            "parent": None,
+            "priorities": {},
+            "tags": [],
+            "user": {
+                "fullname": "Matt Prahl",
+                "name": "mprahl"
+            }
+        }
+        json_two = {
+            "creator": {
+                "fullname": "Tom Brady",
+                "name": "tbrady"
+            },
+            "date_created": "1495035052",
+            "description": "Players of the New England Patriots",
+            "display_name": "Patriots",
```

Comment:
Since this is a trademarked name, we probably should avoid using it. Same for "patriots" on line 373 above.